extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class NarayanaQuarkusConfiguration {
+    // non final properties
+    /**
+     * config
```

Comment:
I assume comment /** config **/ may be omitted.

Reply:
+1